lib/scenic/adapters/postgres.rb

```diff
@@ -81,6 +81,29 @@ def update_view(name, sql_definition)
         create_view(name, sql_definition)
       end
 
+      # Updates a view in the database using `CREATE OR REPLACE VIEW`.
+      #
+      # This results in a `CREATE OR REPLACE VIEW`. Most of the time the
+      # explicitness of the two step process used in {#update_view} is preferred to
+      # `CREATE OR REPLACE VIEW` because the former ensures that the view you are trying
+      # to update did, in fact, already exist. Additionally, `CREATE OR REPLACE
```

Comment:
I think i'd go with the principle of least surprise here. If the name ends up being replace_view (which sounds fine to me) then I'd say giving an error if the view doesn't exist is probably a good thing.